ladybug_geometry/geometry2d/mesh.py

```diff
@@ -64,12 +64,12 @@ def from_dict(cls, data):
         Args:
             data: A python dictionary in the following format
 
-        .. code-block:: json
+        .. code-block:: python
 
             {
             "type": "Mesh2D",
             "vertices": [[0, 0], [10, 0], [0, 10]],
-            "faces": [[0, 1, 2]],
+            "faces": [(0, 1, 2)],
```

Comment:
Good point here. I might recommend using tuples for the `vertices` of the dictionary as well. Since they all need to be iterables with 2 items (one for x and the other for y), using a tuple makes more sense when we're in Python.

Reply:
We should also do this across nearly all of the classes in the library since they all make use of tuples to represent vertex coordinates.